build/conf/bblayers.conf

```diff
@@ -6,6 +6,7 @@ BBPATH = "${TOPDIR}"
 BBFILES ?= ""
 
 BBLAYERS ?= " \
+  ${TOPDIR}/../layers/meta-opentrons \
```

Comment:
Does this mess with recipe/dependency overrides?

Reply:
this just makes sure if there a recipe with the same name, meta-opentron's version of the recipe gets used.